discord_games/wordle.py

```diff
@@ -1,9 +1,9 @@
 from __future__  import annotations
-from typing import Optional
 
 import pathlib
 import random
 import asyncio
+from typing import Optional, Final
```

Comment:
Alvin S, that is the wrong place to import typing. Additionally consider ordering thy imports alphabetically.

Reply:
Jay3332 I know such but I do not want to format the entire thing for Tom the bomb